src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -248,6 +248,8 @@ def _type(value):
 
 name_type = CLIArgumentType(options_list=['--name', '-n'], help='the primary resource name')
 
+edge_zone_type = CLIArgumentType(help='The name of edge zone.', is_preview=True)
```

Comment:
Why call this `edge_zone_type`? This should be called `edge_zone_name` imho

Reply:
this type is CLI argument type, instead of extended location type.

Reply:
Suggestion: add `options_list` to ensure consistency in all command modules.

Reply:
good point! updated.